.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 6
+  - 7
```

Comment:
Node 7 isn't LTS, so it's not maintained anymore and we can remove it.

Reply:
Ok! Just had a look at LTS versions and these are 6, 8, 10. 6 is out, should Travis run through 8 and 10?

Reply:
Yup!

Reply:
Done in 04a0e03a1a9d09bb75427b1862ba94125464f953